crates/alkahest/src/gui/inspector/util.rs

```diff
@@ -208,13 +208,47 @@ impl ComponentPanel for Sphere {
     }
 }
 
+impl ComponentPanel for Cuboid {
+    fn inspector_name() -> &'static str {
+        "Cuboid"
+    }
+
+    fn inspector_icon() -> char {
+        Self::icon().char()
+    }
+
+    fn show_inspector_ui(
+        &mut self,
+        _: &mut Scene,
+        _: &mut Commands<'_, '_>,
+        e: EntityRef<'_>,
+        ui: &mut egui::Ui,
+        _resources: &AppResources,
+    ) {
+        if !e.contains::<Transform>() {
+            ui.label(format!(
+                "{} This entity has no transform component",
+                ICON_ALERT
+            ));
+        }
+
+        ui.horizontal(|ui| {
+            color_edit_button_rgba(ui, &mut self.color, Alpha::OnlyBlend).context_menu(|ui| {
+                ui.checkbox(&mut self.rainbow, "Rainbow mode");
```

Comment:
_i'm beginning to think we need something more generic for the rainbow colors_ 😛